CI.hs

```diff
@@ -17,11 +17,8 @@ import Data.Maybe
 import Data.List.Extra
 import Data.Time.Clock
 import Data.Time.Calendar
-import Data.Semigroup ((<>))
```

Comment:
Maybe time to test this on CI with `-Werror` but let’s leave that for another PR.

Reply:
PR https://github.com/digital-asset/ghc-lib/pull/240 should deal with it (requires rebasing on this).